core-1/upgrades/v2/guide.md

```diff
@@ -0,0 +1,83 @@
+PersistenceCore, core-1 v2 upgrade.
+===
+This on-chain upgrade proposal is for upgrading up Cosmos SDK version from v0.42.4 to v0.44.6. It includes few new modules like authz and feegrant.
+* By voting YES to this proposal, you approve of adding these updates to PersistenceCore.
+* By voting NO to this proposal, you do not approve of this upgrade.
+
+#### Background
+Since the last version of Cosmos SDK there have been a number of updates, fixes and new modules added to the Cosmos SDK, IBC and Tendermint. The following updates are included in persistenceCore v0.2.3:
+* Bump [PersistenceCore](https://github.com/persistenceOne/persistenceCore) to [0.2.3](https://github.com/persistenceOne/persistenceCore/releases/tag/v0.2.3) which notably includes fixes for the vesting accounts and two new modules outlined below. For a detailed difference between PersistenceCore [v0.1.3](https://github.com/persistenceOne/persistenceCore/releases/tag/v0.1.3) and [v0.2.3](https://github.com/persistenceOne/persistenceCore/releases/tag/v0.2.3) check [diff between 0.1.3 and 0.2.3](https://github.com/persistenceOne/persistenceCore/compare/v0.1.3...v0.2.3).
+* Add the [authz](https://github.com/cosmos/cosmos-sdk/tree/v0.44.6/x/authz/spec) module to the Cosmos Hub. \`x/authz\` is an implementation of a Cosmos SDK module, per [ADR 30](https://github.com/cosmos/cosmos-sdk/blob/v0.44.6/docs/architecture/adr-030-authz-module.md), that allows granting arbitrary privileges from one account (the granter) to another account (the grantee). Authorizations must be granted for a particular Msg service method one by one using an implementation of the \`Authorization\` interface.
```

Comment:
```suggestion
* Add the [authz](https://github.com/cosmos/cosmos-sdk/tree/v0.44.6/x/authz/spec) module to the Persistence mainnet. \`x/authz\` is an implementation of a Cosmos SDK module, per [ADR 30](https://github.com/cosmos/cosmos-sdk/blob/v0.44.6/docs/architecture/adr-030-authz-module.md), that allows granting arbitrary privileges from one account (the granter) to another account (the grantee). Authorizations must be granted for a particular Msg service method one by one using an implementation of the \`Authorization\` interface.
```

Reply:
haha this is the old one, so did not change it, just moved it here from the wiki.